samples: wifi: shutdown: Add an option to disable auto-start (#13178)
nrfconnect/sdk-nrf: samples/wifi/shutdown/README.rst

   west build -b nrf7002dk_nrf5340_cpuapp

Disable auto-start of the Wi-Fi driver
--------------------------------------

The Wi-Fi network interface is automatically brought up when the Wi-Fi driver is initialized by default.

> But you can disable this behavior by setting the ``CONFIG_WIFI_INIT_AUTO_START`` configuration option to ``n``.

Reviewer: ```suggestion
This behavior can be disabled by setting the ``CONFIG_WIFI_INIT_AUTO_START`` configuration option to ``n``.
```

> Please use additional configuration file ``overlay-no_wifi_auto_start.conf`` to disable auto-start of the Wi-Fi driver:

Reviewer 2: ```suggestion
Use the additional :file:`overlay-no_wifi_auto_start.conf` configuration file to disable the auto-start of the Wi-Fi driver:
```

.. code-block:: console

>    west build -b nrf7002dk_nrf5340_cpuapp -DEXTRA_CONF_FILE=overlay-no_wifi_auto_start.conf

Reviewer 3: What is the benefit of using a single line overlay instead of passing `-DCONFIG_NRF_WIFI_IF_AUTO_START=n` directly?

Author: Left as a place holder for further configs, but can remove .

> With this configuration, the Wi-Fi network interface is not automatically brought up by the sample.

Reviewer: ```suggestion
With this configuration, the Wi-Fi network interface is not automatically brought up by the nRF70 driver initialization.
```

Author: Well, its the networking stack that brings up the interface, so, `networking stack`

> User should press **Button 1** to bring up the Wi-Fi network interface.

Reviewer 2: ```suggestion
You must press **Button 1** to bring up the Wi-Fi network interface.
```
